terraform/variables.tf

```diff
@@ -0,0 +1,55 @@
+variable "environment" {
+  type        = string
+  description = "the environment the app is running in"
+}
+
+variable "papertrail_destination" {
+  type        = string
+  description = "Where to send logs hostname:port"
+}
+
+variable "secret_key_base" {
+  type        = string
+  description = "secret key base for rails apps"
+}
+
+variable "auth0_client_id" {
+  type        = string
+  description = "AUTH0 Client ID"
+}
+
+variable "auth0_client_secret" {
+  type        = string
+  description = "AUTH0 Client Secret"
+}
+
+variable "auth0_domain" {
+  type        = string
+  description = "AUTH0 domanin"
+}
+
+variable "notify_key" {
+  type        = string
+  description = "Notify key"
+}
+
+variable "notify_welcome_email_template" {
+  type        = string
+  description = "Notify welcome email template"
+}
+
+variable "rollbar_access_token" {
+  type        = string
+  description = "Rollbar access token"
+}
+
+variable "docker_image" {
+  type        = string
+  description = "docker image to use"
+  default     = "develop"
```

Comment:
I currently wondering if we want a default here and if we do is `develop` the right one

Reply:
If this value determines what gets deployed if a variable isn't provided, I think it should default to the safest stable version which should be `master`.

There would be less impact in staging being set back to master, than pushing master forward unexpectedly to the latest changes (which might not be stable).

Reply:
Indeed. I'll change it to master.

Reply:
Thinking about this I think it makes sense for it to not have a default.